charts/gitops-runtime/values.yaml

```diff
@@ -162,7 +163,59 @@ argo-cd:
     enabled: true
     # -- Extract application version based on ApplicationConfiguration CRD
     useApplicationConfiguration: true
-  notifications: {}
+  notifications:
+    enabled: true
```

Comment:
Is it a good idea to have something enabled by default?
Is it a essential component?

Reply:
the notifications are what triggers the promotion flows initially. it's part of the solution.